src/Symfony/Component/Console/Tests/ApplicationTest.php

```diff
@@ -914,7 +914,7 @@ public function testRunWithDispatcher()
     }
 
     /**
-     * @expectedException        \LogicException
+     * @expectedException        \RuntimeException
```

Comment:
oh now I get it! I've added a little test for that too. Hope everything is ok now :)